modules/platforms/cpp/thin-client/include/ignite/thin/ignite_client_configuration.h

```diff
@@ -230,6 +231,29 @@ namespace ignite
                 return partitionAwareness;
             }
 
+            /**
+             * Get connection limit.
+             * Zero means that number of active connections is not limited.
```

Comment:
What if I set `-1`?

Reply:
But it is unsigned. If you set -1 compiler will give you either a warning or error depending on your compiler flags.

Reply:
Oops, did not pay attention to the type.